tutorials/plot_background_filtering.py

```diff
@@ -483,15 +524,232 @@ def plot_signal(x, offset):
 plt.show()
 
 ###############################################################################
-# Filtering in MNE-Python
-# =======================
+# Some pitfalls of filtering
+# ==========================
+# Multiple recent papers have noted potential risks of drawing
+# errant inferences due to misapplication of filters.
+#
+# Low-pass problems
+# -----------------
+# Filters in general, especially those that are acausal (zero-phase), can make
+# activity appear to occur earlier or later than it truly did. As
+# mentioned in VanRullen 2011 [3]_, investigations of commonly (at the time)
+# used low-pass filters created artifacts. However, such deleterious effects
+# of low-passing were found to have minimal impact for many real-world
+# examples in Rousselet 2012 [5]_.
+#
+# Perhaps more revealing, it was noted in Widmann & Schröger 2012 [6]_ that
+# the problematic low-pass filters from VanRullen 2011 [3]_:
+#
+#    1. Used a least-squares design (like :func:`scipy.signal.firls`) that
+#       included "do-not-care" transition regions, which can lead to
+#       uncontrolled behavior.
+#    2. Filter length that was independent of the transition bandwidth,
+#       which can cause excessive ringing and signal distortion.
+#
+# .. _tut_filtering_hp_problems:
+#
+# High-pass problems
+# ------------------
+# When it comes to high-pass filtering, using corner frequencies above 0.1 Hz
+# were found in Acunzo *et al.* 2012 [4]_ to:
+#
+#    "...generate a systematic bias easily leading to misinterpretations of
+#    neural activity.”
+#
+# In a related paper, Widmann *et al.* 2015 [7]_ also came to suggest a 0.1 Hz
+# highpass. And more evidence followed in Tanner *et al.* 2015 [8]_ of such
+# distortions. Using data from language ERP studies of semantic and syntactic
+# processing (i.e., N400 and P600), using a high-pass above 0.3 Hz caused
+# significant effects to be introduced implausibly early when compared to the
+# unfiltered data. From this, the authors suggested the optimal high-pass
+# value for language processing to be 0.1 Hz.
```

Comment:
Could also note that it depends on the nature of effects: in sentence processing stuff like here, you get very late, low-freq components (like the P600 [which is in fact just a P3]). If you look ONLY at early visual components, there may not be much of a difference between .1 and 1 Hz, although if you ALSO have a P3 in the data, it will smear into the data.

Generally, the closer the filter is to the actual signal, the more problems can occur.